tensorboard/program.py

```diff
@@ -526,7 +526,7 @@ def get_url(self):
       host = self._flags.host
       display_host = (
           '[%s]' % host if ':' in host and not host.startswith('[') else host)
-    return 'http://%s:%d%s' % (display_host, self.server_port,
+    return 'http://%s:%d%s/' % (display_host, self.server_port,
```

Comment:
Okay, so now you just need to fix the fact this still appends a redundant second `/` when path_prefix contains a slash.  This is why the Travis check is failing in the second assertion in testPathPrefixSlash(), as shown:

```text
FAIL: testPathPrefixSlash (__main__.WerkzeugServerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/.bazel-output-base/sandbox/linux-sandbox/4/execroot/org_tensorflow_tensorboard/bazel-out/k8-fastbuild/bin/tensorboard/program_test.runfiles/org_tensorflow_tensorboard/tensorboard/program_test.py", line 76, in testPathPrefixSlash
    self.assertEndsWith(server.get_url(), '/test/')
  File "/home/travis/.bazel-output-base/sandbox/linux-sandbox/4/execroot/org_tensorflow_tensorboard/bazel-out/k8-fastbuild/bin/tensorboard/program_test.runfiles/org_tensorflow_tensorboard/tensorboard/test.py", line 92, in assertEndsWith
    self.fail(fail_msg)
AssertionError: 'http://travis-job-c803f8d6-a50b-4bd6-924a-5036a813e700:38810/test//' does not end with '/test/'
```

You could do this by just changing this string format call to take `self._flags.path_prefix.rstrip('/')`.

Reply:
Done! @nfelt Please See

Reply:
This didn't fix the test because rstrip() returns a new string.  Python strings are immutable so calling it on a separate line by itself does nothing.

You need to change the next line where `self._flags.path_prefix` is passed into the string formatting call.

Reply:
Did that! 
@nfelt  now look!